cheroot/server.py

```diff
@@ -1862,19 +1862,22 @@ def prepare_socket(bind_addr, family, type, proto, nodelay, ssl_adapter):
         """Create and prepare the socket object."""
         sock = socket.socket(family, type, proto)
         prevent_socket_inheritance(sock)
-        if not IS_WINDOWS:
+
+        host, port = bind_addr[:2]
+
+        if not IS_WINDOWS and port != 0:
```

Comment:
It would be better readable if this expression was named. i.e. `IS_EPHEMERAL_PORT = port == 0`